apps/server/src/app/__tests__/balance-sync.integration.spec.ts

```diff
@@ -130,7 +130,7 @@ describe('balance sync strategies', () => {
                                     amount: 100,
                                     quantity: 10,
                                     price: 10,
-                                    category: InvestmentTransactionCategory.buy,
+                                    plaidType: 'buy',
```

Comment:
My plan was to have both `plaidType` & `category` here

Reply:
can't set category until to generated expression is dropped from the db which I think is in your changes. I'll need to go back in and re-do all the stuff I rolled back after your PR is merged

Reply:
Hmmm too bad that's not possible 😓

Reply:
You can make your changes which will break this test temporarily and then we can go back in and rework the test to use category directly rather than relying on the plaidTypes/subTypes. Think that's the only way to do it unless you want to make the test fixes as part of removing all the plaid stuff

Reply:
I'll be ready and waiting for the change to land, I've already made the changes before so it should be a quick fix after I merge in your changes.